app/assets/javascripts/hyrax/admin/analytics_stats.js

```diff
@@ -0,0 +1,25 @@
+Blacklight.onLoad(function() {
```

Comment:
Rather than doing `Blacklight.onLoad`, would you mind adding something in https://github.com/samvera/hyrax/blob/master/app/assets/javascripts/hyrax/app.js.erb this is the newer way we're trying to initialize all of the javascript in the Hyrax application.